### DISCUSSION TITLE
Create a css file to apply style globally

### DISCUSSION THREAD

**Discussion by price76** [befrooshim/com.befrooshim.window#14] [Ideas]
To improve style code, normalize it and reduce repetition, create main css file is a appropriate way. So its good to open a issue for this task.

**Comment by mohsenxad** [comment]
👌 that's great

We can use **reset css boiler plates**, that exists. Check [this link](https://gist.github.com/dkanderson/5879765)

If it's ok. create a issue for this implementation.
